code/game/objects/items/robot/robot_items.dm

```diff
@@ -96,6 +96,12 @@
 		if(1)
 			if(M.health >= 0)
 				if(ishuman(M))
+					M.AdjustStun(-60)
+					M.AdjustKnockdown(-60)
+					M.AdjustUnconscious(-60)
+					M.AdjustSleeping(-100)
+					M.AdjustParalyzed(-60)
+					M.AdjustImmobilized(-60)
```

Comment:
Could you turn this into a proc that both this and the  carbon `help_shake_act()` call, instead of copypasting the code?
Makes it easier to maintain it, in case someone changes the values in the future.

Reply:
I am still learning how to code, so that might be a challenge for me, that will take some time. I am determined to get this done though.